cloudformation/policies/parallelcluster-policies.yaml

```diff
@@ -54,6 +54,10 @@ Parameters:
     AllowedValues:
       - true
       - false
+  IAMRoleAndPolicyPrefix:
+    Type: String
+    Description: 'Prefix for IAM roles and policies'
+    Default: ''
```

Comment:
We should reflect here the same parameter constraint used in the other template.